### ISSUE TITLE
Normalization for S1?

### ISSUE THREAD

**Issue by bengmstrong** [zhu-xlab/SSL4EO-S12#17]
I'd like to use one of your pre-trained Sentinel-1 models, but I don't see how you normalized your Sentinel-1 data. In the readme, I see the normalization scheme for Sentinel-2 ("input clip to [0,1] by dividing 10000"), but can you add instructions for how to pre-process Sentinel-1?

**Comment by wangyi111** [comment]
Hi! Yes we used dB scale images, removed extreme values, and normalized mean and standard deviation for each channel.

**Comment by bengmstrong** [comment]
great thanks! So just to clarify:
For S1 data, you removed extreme values and then took Mean / Std  and then normalized like 
$\bar{X} = \frac{X - \mu}{\sigma}$

which gives a normalized range of values that is both positive and negative, with mean equal to zero. Or did you use the normalization function that's in the code [here](https://github.com/zhu-xlab/SSL4EO-S12/blob/bc0454331f627c248b505423b495adc12558838d/src/benchmark/pretrain_ssl/datasets/SSL4EO/ssl4eo_dataset.py#L36), which results in values clipped to 0-255?

For S2 data, you simply divided by 10000, resulting in values in the range [0,1]

Is that right?

**Comment by wangyi111** [comment]
Hi! Yes for the pretraining we used the normalization in the code (values clipped to 0-255), the input to the network after dataloader is squeezed to 0-1.

For S2, we simply divided by 10k in pretraining.

Note that in downstream tasks, both "0-1" and "0 centered" normalization should work. In our experience, using exactly the same as the pretraining not necessarily always leads to the best performance.
